android/app/src/main/java/org/mozilla/magnet/magnetapi/ApiMagnet.java

```diff
@@ -16,5 +16,6 @@ public ApiMagnet(Context context) {
         mount("channels", new ApiChannels(context));
         mount("subscriptions", new ApiSubscriptions(context));
         mount("metadata", new ApiMetadata(context));
+        mount("preferences", new ApiPreferences(context));
```

Comment:
@samgiles off topic a little: Interested how you imagine these user-state routes eventually persisting to magnet-service, are they in an alright state to do so?